aten/src/ATen/native/native_functions.yaml

```diff
@@ -8572,7 +8575,8 @@
 - func: adaptive_avg_pool2d.out(Tensor self, int[2] output_size, *, Tensor(a!) out) -> Tensor(a!)
   python_module: nn
   dispatch:
-    CPU, CUDA: adaptive_avg_pool2d_out_cpu
+    CPU: adaptive_avg_pool2d_out_cpu
+    CUDA: adaptive_avg_pool2d_out_cuda
```

Comment:
this isn't strictly related to this diff but i was in the neighborhood and noticed this - i think it's a bug since `adaptive_avg_pool2d_out_cuda` exists.

Reply:
It was changed in https://github.com/pytorch/pytorch/pull/40182/ and it does look like a bug to me. Good catch!

Reply:
O_O this is awesome!